src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx

```diff
@@ -81,12 +82,18 @@ const BlastInputSequencesHeader = (props: Props) => {
     ? sequences.length + 1
     : sequences.length;
 
+  const sequenceCounterClass = classNames(styles.sequenceCounter, {
+    [styles.sequenceCounterError]: sequences.length > MAX_BLAST_SEQUENCE_COUNT
+  });
```

Comment:
Why do we need this when the `Add another sequence` button is disabled when the maximum sequence count reaches 50?

Reply:
we discussed this in the daily scrum this morning, you might have missed it. I confirmed this with Andrea and Lucy, its just another visual indication of whats wrong.

Reply:
Yes but this isn't shown at all right? I tried adding 50 sequences but there was no visual indicator to show this.

<img width="1018" alt="Screenshot 2022-05-19 at 17 50 59" src="https://user-images.githubusercontent.com/32512909/169355113-458c9125-12c1-4f5b-a1b2-4a21be52cbca.png">